### ISSUE TITLE
sql: ERROR: type pg_catalog.int8 does not exist

### ISSUE THREAD

**Issue by mgartner** [cockroachdb/cockroach#57971]
**Describe the problem**

I went through the [TypeORM Quick Start](https://typeorm.io/#/undefined/quick-start). When running the generated app with `npm start`, a `user` table is successfully created. However, if you run the app again (`npm start`), TypeORM queries `information_schema` and gets an error: `ERROR: type 'pg_catalog"."int8' does not exist`.

**To Reproduce**

You can follow the [TypeORM Quick Start](https://typeorm.io/#/undefined/quick-start) and run `npm start` twice in a row.

Or you can simulate what TypeORM is doing by manually creating the table and running the problematic query:

```
CREATE TABLE public."user" (
  id INT8 NOT NULL DEFAULT unique_rowid(),
  "firstName" VARCHAR NOT NULL,
  "lastName" VARCHAR NOT NULL,
  age INT8 NOT NULL,
  CONSTRAINT "PK_cace4a159ff9f2512dd42373760" PRIMARY KEY (id ASC),
  FAMILY "primary" (id, "firstName", "lastName", age)
);


SELECT
  *,
  pg_catalog.col_description(
    ('"' || table_catalog || '"."' || table_schema || '"."' || table_name || '"')::regclass::oid,
    ordinal_position
  ) as description,
  ('"' || "udt_schema" || '"."' || "udt_name" || '"')::"regtype" AS "regtype"
FROM "information_schema"."columns"
WHERE ("table_schema" = 'public' AND "table_name" = 'user');
```

```
ERROR: type 'pg_catalog"."int8' does not exist
SQLSTATE: 42704
```

**Expected behavior**

There should be no error.

**Environment:**

```
Build Tag:        v20.2.1
Build Time:       2020/11/20 18:37:02
Distribution:     CCL
Platform:         darwin amd64 (x86_64-apple-darwin14)
Go Version:       go1.13.14
C Compiler:       4.2.1 Compatible Clang 3.8.0 (tags/RELEASE_380/final)
Build Commit ID:  fe9afecdaf3ffc41c6822897595e85451dad26f2
Build Type:       release
```

**Comment by mgartner** [comment]
cc @rafiss

**Comment by rafiss** [comment]
This is fixed by https://github.com/cockroachdb/cockroach/pull/55607

We didn't backport it but if TypeORM is broken without it that would be a more compelling reason to backport. (Oliver is on PTO so not cc-ing him at the moment)

**Comment by rafiss** [comment]
Hmm actually I can't repro this in crdb 20.2 and using typeorm 0.2.29.

I'm not sure why, since I see the bad query here (https://github.com/typeorm/typeorm/commit/bc623a42a868eae7c988779abc4cdc0bbf775def#diff-3af60f660db25bba92d1aadb6b0d86dfad4cc1e4f0061f8b767b66c3c512b05d) and when I run it manually it doesn't work.

Anyway, our team decided we can't backport the fix since the problem is pre-existing to 20.2 and the change is too complicated. So unfortunately I will close this without resolving for 20.2 (but it's fixed for 21.1).
